TestZMQ.md

```diff
@@ -0,0 +1,25 @@
+# <a name="heading"></a>ZeroMQ for Synchronous Work Order processing
+
+
+This module uses ZeroMQ sockets.ZeroMQ is a library used to implement messaging and communication systems between applications and processes.This module uses the Request/Reply Pattern for socket communication to enable synchronous workorder execution.
+
+# <a name="testing"></a>Testing ZeroMQ
+
+Once the code is successfully built, you could run the below test steps to check if the
+installation is working correctly.
+Follow these steps to run the `Demo.py` testcase:
+1. For Docker-based builds:
+   1. Follow the steps from BUILD.md for
+      ["Docker-based Build and Execution"](#dockerbuild)
+   2. Terminal 1 is running `docker-compose` and Terminal 2 is running the
+      "tcf" Docker container shell
+2. In Terminal 2, run `cd $TCF_HOME/tests`
+3. In Terminal 2, use this command to run the `Demo.py` test:
+   ```bash
+   python3 Demo.py --input_dir ./json_requests/ \
+           --connect_uri "http://localhost:1947" work_orders/output.json
```

Comment:
We will be deprecating demo.py soon. Please add the documentation for testing w.r.t generic client -> https://github.com/hyperledger/avalon/tree/master/examples/apps/generic_client

Reply:
Please review the latest changes - it could be tested with both generic_client and Demo.py.